convert2rhel/subscription.py

```diff
@@ -89,17 +96,31 @@ def register_system():
     attempt = 0
     while True and attempt < MAX_NUM_OF_ATTEMPTS_TO_SUBSCRIBE:
         registration_cmd = get_registration_cmd()
-        loggerinst.info("Attempt %s of %s: Registering system by running subscription-manager"
-                        " command ... ", attempt+1, MAX_NUM_OF_ATTEMPTS_TO_SUBSCRIBE)
+
+        if attempt > 0:
+            loggerinst.info(
+                "Attempt %s of %s: Registering system by running subscription-manager"
+                " command ... ",
+                attempt + 1,
+                MAX_NUM_OF_ATTEMPTS_TO_SUBSCRIBE,
+            )
+        else:
+            loggerinst.info(
+                "Registering system by running subscription-manager" " command ... "
+            )
```

Comment:
I'd switch this around so we have an `attempt_msg`, that way we reduce the duplication of the command (could also have a string for `Registering system by running subscription-manager command ...`, but this solution seems more readable to me). 

Also had two quotes in the logger
`"Registering system by running subscription-manager command"`
vs
`"Registering system by running subscription-manager" " command"`
```suggestion
		attempt_msg = ""
        if attempt > 0:
            attempt_mgs = "Attempt %d of %d: " % (attempt + 1, MAX_NUM_OF_ATTEMPTS_TO_SUBSCRIBE)
        loggerinst.info(
            "%sRegistering system by running subscription-manager command ...", attempt_mgs
        )
```